### PR TITLE
Updates for Dojo v4.0.0

### PR DESCRIPTION
**Pull request by agubler**
**Type:** feature

The following has been addressed in the PR:

* [ ] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [x] Unit or Functional tests are included in the PR

**Description:**

Updates widgets and examples to use dojo v4.0.0 (currently alpha)

### CODE REVIEW

File: src/time-picker/example/index.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { getDateFormatter } from '@dojo/framework/i18n/date';
 import { DNode } from '@dojo/framework/widget-core/interfaces';
-import { ProjectorMixin } from '@dojo/framework/widget-core/mixins/Projector';
+import renderer from '@dojo/framework/widget-core/vdom';
 import { theme, ThemedMixin, ThemedProperties } from '@dojo/framework/widget-core/mixins/Themed';
 import { v, w } from '@dojo/framework/widget-core/d';
 import { WidgetBase } from '@dojo/framework/widget-core/WidgetBase';
 import setLocaleData from './setLocaleData';
-import TimePicker, { getOptions, TimeUnits } from '../../time-picker/index';
+import TimePicker, { getOptions, TimeUnits } from '../../time-picker/index'
```
Comment (smhigley) [inline src/time-picker/example/index.ts]: is this intentional?

Comment (agubler) [reply src/time-picker/example/index.ts]: nope 😄 👍

**Comment by smhigley** [comment]
Would you mind switching the grid example page from `ProjectorMixin` to `renderer` while fixing the merge conflicts 🙏? Other than that, I'm ready to approve.

**Comment by agubler** [comment]
@smhigley updated 😄

**Comment by smhigley** [comment]
The toolbar example page looks a bit off with the new renderer, and keeps the toolbar-specific styles inserted after navigating to a different widget.

**Comment by agubler** [comment]
@smhigley toolbar had some weird styles added when the example was attached.... I have removed them.

**Pull request merged**